modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/CppRestbedServerCodegen.java

```diff
@@ -301,6 +304,8 @@ public String toDefaultValue(Schema p) {
                 return "0L";
             }
             return "0";
+        } else if (ModelUtils.isByteArraySchema(p)) {
```

Comment:
Shouldn't it be the same like for string schema?
```C++
         if (ModelUtils.isStringSchema(p)) {
             return "\"\"";
```

Reply:
yes, you are right. i copied it over from pistache. so pistache seems to also have an error then.